UACloudLibraryServer/Interfaces/IDatabase.cs

```diff
@@ -67,18 +65,29 @@ public interface IDatabase
         IQueryable<ObjectModel> GetObjects(string nodeSetUrl = null, DateTime? publicationDate = null, string nodeId = null);
         IQueryable<NodeModel> GetAllNodes(string nodeSetUrl = null, DateTime? publicationDate = null, string nodeId = null);
 
-        Task<List<Category>> GetCategory(int limit, int offset, string where, string orderBy);
-        Task<List<Organisation>> GetOrganisation(int limit, int offset, string where, string orderBy);
+        IQueryable<CategoryModel> GetCategories();
+        IQueryable<OrganisationModel> GetOrganisations();
 
-        IQueryable<MetadataModel> GetMetadataModel();
-        Task<List<UANameSpace>> GetNamespaces(int limit, int offset, string where, string orderBy);
+        IQueryable<NamespaceMetaDataModel> GetNamespaces();
         int GetNamespaceTotalCount();
+
+        Task<UANameSpace> ApproveNamespaceAsync(string identifier, ApprovalStatus status, string approvalInformation);
+        IQueryable<CloudLibNodeSetModel> GetNodeSetsPendingApproval();
+
 #if !NOLEGACY
```

Comment:
Leave it in there but set the NOLEGACY compiler flag so we can bring it back easily, if required.